### PR TITLE
19553 remove payments feature toggle

### PR DESCRIPTION
**Pull request by micahchiang**
## Description
This pull request removes the feature toggle code for view payments history, as it's no longer needed. 

## Screenshots
<details><summary>CTA Widget</summary>

Screen Shot 2021-02-16 at 11 55 27 AM
Screen Shot 2021-02-16 at 11 56 27 AM

</details>

## Testing done
- local

## Acceptance criteria
- [x] tests pass
- [x] CTA works without feature toggle

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by Mottie — APPROVED** [review]
ghe-emoji

**Review by jhonny218 — APPROVED** [review]

File: src/applications/static-pages/view-payment-history/ViewPaymentHistoryCTA.js [hunk 1]
```diff
@@ -90,7 +57,6 @@ const ViewPaymentHistoryCTA = props => {
 };
 
 const mapStateToProps = store => ({
-  includedInFlipper: toggleValues(store)[FEATURE_FLAG_NAMES.viewPaymentHistory],
```
Comment (ncksllvn) [inline src/applications/static-pages/view-payment-history/ViewPaymentHistoryCTA.js:93]: Can we go a step further by removing this widget in favor of a [call-to-action widget](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/site-wide/cta-widget/helpers.js#L11)? If you aren't familiar, the green alertbox [here](https://www.va.gov/change-direct-deposit/) is an example

Comment (micahchiang) [reply src/applications/static-pages/view-payment-history/ViewPaymentHistoryCTA.js:93]: Can do!

**Review by cohnjesse — APPROVED** [review]
LGTM

**Pull request merged**